I can't reproduce a working tox run for py38:
```
========================================= ERRORS =========================================
_____________________________ ERROR collecting test session ______________________________
/usr/lib64/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
.tox/py38/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:168: in exec_module
    exec(co, module.__dict__)
flaat/flask/conftest.py:4: in <module>
    from pytest_cases import fixture
E   ModuleNotFoundError: No module named 'pytest_cases'
================================ short test summary info =================================
ERROR  - ModuleNotFoundError: No module named 'pytest_cases'
!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!
==================================== 1 error in 0.26s ====================================
ERROR: InvocationError for command /home/marcus/projects/flaat/.tox/py38/bin/pytest (exited with code 2)
________________________________________ summary _________________________________________
ERROR:   py38: commands failed
```
It's the same error as on py39 and py310.

--- a/doc/source/index.rst
+++ b/doc/source/index.rst
@@ -19,6 +19,7 @@ currently supported:
 
    flaat/installation
    flaat/examples/aio
+   flaat/examples/flask
    flaat/flaat-userinfo
    flaat/development
 

--- a/flaat/flask/__init__.py
+++ b/flaat/flask/__init__.py
@@ -120,7 +120,7 @@ class Flaat(BaseFlaat):
         current_app.config["FLAAT_VERIFY_TLS"] = verify_tls
 
     @property
-    def verify_tls(self):
+    def verify_jwt(self):
         """Skip JWT verification while processing requests.
         :return: Boolean, indicates if TLS is enabled
         """

